kodeverk/src/main/java/no/nav/ung/kodeverk/behandling/BehandlingÅrsakType.java

```diff
@@ -91,6 +91,10 @@ public enum BehandlingÅrsakType implements Kodeverdi {
     RE_RAPPORTERING_INNTEKT("RE-RAPPORTERING-INNTEKT", "Rapportering av inntekt"),
     RE_KONTROLL_REGISTER_INNTEKT("RE-KONTROLL-REGISTER-INNTEKT", "Kontroll av registerinntekt"),
 
+    // Generell oppgavebekreftelse
+    UTTALELSE_FRA_BRUKER("UTTALELSE-FRA-BRUKER", "Uttalelse fra bruker"),
```

Comment:
et det mulig i dag?

Reply:
generell uttalelse fra bruker... uten etterlysning ?

Reply:
Nei, vil alltid skje sammen med etterlysning i dag

Reply:
Dette er for å trigge ein diff. Alternativet hadde vore å hatt eit etterlysningsrgrunnlag/uttalelsegrunnlag og diffet dette. Tenkjer du det er bedre?